docs/code_ref/coordinates/rotatedsunframe.rst

```diff
@@ -201,7 +201,7 @@ We specify a range of durations from -5 days to +5 days, stepping at 1-day incre
 
 Let's convert to the base coordinate frame to reveal the motion of the active region over time::
 
-  >>> ar.transform_to(ar.base)
+  >>> ar.transform_to(ar.base)  # doctest: +FLOAT_CMP
```

Comment:
I thought we have this set in setup.cfg?

Reply:
Thanks for reminding - this change didn't do anything anyway, because the differences between different CI builds are greater than floating point precision.